pyccel/parser/semantic.py

```diff
@@ -2426,7 +2443,8 @@ def _visit_FunctionDef(self, expr, **settings):
             body = self._visit(expr.body)
 
             # ISSUE 177: must update arguments to get is_target
```

Comment:
@EmilyBourne @saidctb This comment refers to an issue that was closed a long time ago, does it still carry a meaning?

Reply:
I don't think so